layout/page.swig

```diff
@@ -1,17 +1,7 @@
 {% extends '_layout.swig' %}
 {% import '_macro/sidebar.swig' as sidebar_template %}
 
-{% block title %}
-  {% set page_title_suffix = ' | ' + config.title %}
-
-  {% if page.type === "categories" and not page.title  %}
-    {{ __('title.category') + page_title_suffix }}
-  {% elif page.type === "tags" and not page.title  %}
-    {{ __('title.tag') + page_title_suffix }}
-  {% else %}
-    {{ page.title + page_title_suffix }}
-  {% endif %}
-{% endblock %}
+{% block title %}{% set page_title_suffix = ' | ' + config.title %}{% if page.type === "categories" and not page.title %}{{ __('title.category') + page_title_suffix }}{% elif page.type === "tags" and not page.title %}{{ __('title.tag') + page_title_suffix }}{% else %}{{ page.title + page_title_suffix }}{% endif %}{% endblock %}
```

Comment:
But easy to read for search engines. For them it's will `/n/r &nbsp` in Titles. It's very bad.
I wrote about it in subcomment.